### PR TITLE
SearchBar component pushed

### PR DESCRIPTION
**Pull request by ShubhamBade**
This branch has search bar component code. Thank you!!

### CODE REVIEW

File: src/components/searchBar/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { SearchBarInterface } from '../../interface/searchBar/SearchBarInterface';
+
+export const SearchBar : React.FC<SearchBarInterface & { onChangeText: (text: string) => void }> = ({
+    placeHolder = "Search here",
+    backgroundColor = "#1F242E",
+    mode = "outlined",
+    width = 90,
+    activeOutlineColor = "#E4E4E4",
+    outlineColor = "#3D3E41",
```
Comment (rohan-arya-w) [inline src/components/searchBar/SearchBar.tsx]: Don't use Hardcoded color value

**Review by rohan-arya-w — CHANGES_REQUESTED** [review]
Remove all Hardcoded values for colors

File: src/components/searchBar/index.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export {SearchBar} from './SearchBar'
+
+// export all types of buttons here
```
Comment (rohan-arya-w) [inline src/components/searchBar/index.ts]: Remove the comment write something meaning full here

**Review by rohan-arya-w — CHANGES_REQUESTED** [review]

File: src/components/searchBar/SearchBar.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    textColor = "#687797",
+    borderRadius = 2,
+    error = false,
+    errorMessage = '* Error: Payment Source is required',
```
Comment (tabassum2507) [inline src/components/searchBar/SearchBar.tsx]: Do we need error message in Search bar?

**Review by tabassum2507 — APPROVED** [review]

**Review by rohan-arya-w — APPROVED** [review]

**Review by emailpocketxtra — APPROVED** [review]

**Pull request merged**